### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**

### CODE REVIEW

File: absl/tests/app_test.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def test_usage(self):
       app.usage()
     self.assertIn(__doc__, mock_stderr.getvalue())
     # Assert that flags are written to stderr.
-    self.assertIn('\n  --[no]helpfull:', mock_stderr.getvalue())
+    self.assertIn('\n  --[no]helpful:', mock_stderr.getvalue())
```
Comment (yilei) [inline absl/tests/app_test.py:75]: This is not a typo, this "help" "full" as opposed to "help" "short"

**Review by yilei — CHANGES_REQUESTED** [review]
Thank you!

File: absl/tests/app_test.py (hunk 1 above)
Comment (kianmeng) [reply absl/tests/app_test.py]: I see. Got it.

**Review by yilei — APPROVED** [review]

**Comment by kianmeng** [comment]
:tada: :tada: :tada: :tada: :tada:

**Pull request merged**